virttest/utils_misc.py

```diff
@@ -1248,13 +1248,13 @@ def create_x509_dir(path, cacert_subj, server_subj, passphrase,
     cmd_set = [
         ('%s genrsa -des3 -passout pass:%s -out %sca-key.pem %d' %
          (ssl_cmd, passphrase, path, bits)),
-        ('%s req -new -x509 -days %d -key %sca-key.pem -passin pass:%s -out '
+        ('%s req -new -x509 -days %d -key %sca-key.pem -passin pass:%s -passout pass:%s -out '
```

Comment:
Will this change keep compatible with previous openssl version?

Reply:
Yes, this change can run on RHEL 8.